### ISSUE TITLE
How to change CSS dynamically in Angular-Calendar angular2

### ISSUE THREAD

**Issue by akshayextentia** [mattlewis92/angular-calendar#89]
Hi Matt,

I like your plugin angular-calendar with typescript in Angular2. it was awesome. I am using this in my project. 
I have problem with css, How to change CSS dynamically as per my requirement for Day view and week view. I found the grid view in day and week view which is good but my requirement is getting row wise data as well as if we found same date and time then row will be divided in to number of data. 

Basically i found all functionality in angular1.X calendar but i want same thing in angular2.
I want some functionality which i can use it.
Plugin in my calendar :  "angular-calendar": "^0.4.0" 
Please help me in that

**Comment by mattlewis92** [comment]
If you need a more advanced week view than what this library provides I'd suggest just building your own as it's impossible to provide a calendar that meets everyones use cases. One of the major changes I made when rewriting this lib for angular2 was to provide split components for the month / week / day view. If one view doesn't work well for your requirements then you can simply write your own and swap it out. The `calendar-utils` module is also split out so you can use that lib to handle most of the business logic for building your own view.
